kloppy/_providers/signality.py

```diff
@@ -0,0 +1,56 @@
+from contextlib import ExitStack
+from typing import Optional, Iterable, List
+
+from kloppy.domain import TrackingDataset
+from kloppy.infra.serializers.tracking.signality import (
+    SignalityDeserializer,
+    SignalityInputs,
+)
+from kloppy.io import FileLike, open_as_file, expand_inputs
+
+
+def load(
+    meta_data: FileLike,
+    raw_data_feeds: Iterable[FileLike],
+    venue_information: FileLike,
+    sample_rate: Optional[float] = None,
+    limit: Optional[int] = None,
+    coordinates: Optional[str] = None,
+) -> TrackingDataset:
+    """
+    Load and deserialize tracking data from multiple input files.
+
+    Args:
+        meta_data (FileLike): File-like object containing metadata information of the game.
+        raw_data_feeds (Iterable[FileLike]): An iterable of file-like objects containing raw tracking data feeds.
+        venue_information (FileLike): File-like object containing venue information where the game was played.
+        sample_rate (Optional[float]): Sampling rate to be applied during deserialization (default: None).
+        limit (Optional[int]): Limit on the number of data points to process (default: None).
```

Comment:
What is a "data point" here: a single player's position or a frame? Better write "limit the number of frames loaded"